docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfigurator.java

```diff
@@ -61,18 +64,22 @@ public void configure(@Observes CubeConfiguration event, ArquillianDescriptor ar
     }
 
     private void configure(ArquillianDescriptor arquillianDescriptor) {
-        operatingSystemFamilyInstanceProducer.set(new OperatingSystemResolver().currentOperatingSystem().getFamily());
-        Map<String, String> config = arquillianDescriptor.extension(EXTENSION_NAME).getExtensionProperties();
-        CubeDockerConfigurationResolver resolver = new CubeDockerConfigurationResolver(topInstance.get(),
-            dockerMachineInstance.get(),
-            boot2DockerInstance.get(),
-            operatingSystemFamilyInstanceProducer.get());
-        resolver.resolve(config);
-        CubeDockerConfiguration cubeConfiguration = CubeDockerConfiguration.fromMap(config, injectorInstance.get());
-        cubeConfiguration = resolveDynamicNames(cubeConfiguration);
-        System.out.println(cubeConfiguration);
-        hostUriContextInstanceProducer.set(new HostIpContext(cubeConfiguration.getDockerServerIp()));
-        configurationProducer.set(cubeConfiguration);
+        try {
+            operatingSystemFamilyInstanceProducer.set(new OperatingSystemResolver().currentOperatingSystem().getFamily());
+            Map<String, String> config = arquillianDescriptor.extension(EXTENSION_NAME).getExtensionProperties();
+            CubeDockerConfigurationResolver resolver = new CubeDockerConfigurationResolver(topInstance.get(),
+                dockerMachineInstance.get(),
+                boot2DockerInstance.get(),
+                operatingSystemFamilyInstanceProducer.get());
+            resolver.resolve(config);
+            CubeDockerConfiguration cubeConfiguration = CubeDockerConfiguration.fromMap(config, injectorInstance.get());
+            cubeConfiguration = resolveDynamicNames(cubeConfiguration);
+            System.out.println(cubeConfiguration);
+            hostUriContextInstanceProducer.set(new HostIpContext(cubeConfiguration.getDockerServerIp()));
+            configurationProducer.set(cubeConfiguration);
+        } catch (ExecutionException e) {
+            log.log(Level.WARNING, "Error executing docker command", e);
```

Comment:
Yes I understand your use case, but an exception is going to be thrown anyway of your code right? At the end not throwing an exception at this point does not imply that test is skipped just that Docker Client is going to be executed and would not find the docker host.

In case of `RequiresDocker` it might have sense because it will avoid running that test, but in this case another runtime exception will be thrown. For example instead of throwing an `ExecutionException`  exception, a NPE or ConnectionRefused exception are going to be thrown.